backend/src/test/java/de/neuefische/backend/controller/JobControllerTest.java

```diff
@@ -25,4 +32,183 @@ void getAllJobs_ShouldReturnEmptyList() throws Exception {
+                            "startTime": "10:00:00",
+                            "endTime": "12:00:00",
+                            "status": null,
+                            "jobComment": "Job Comment"
+                        }
+                    """))
+                .andExpect(status().isCreated())
+                .andReturn();
+        String content = result.getResponse().getContentAsString();
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        Job[] addedJobs = objectMapper.readValue(content, Job[].class);
+        Job addedJob = addedJobs[0];
+
+        String updatedJobJson = """
+    {
+        "jobFormat": "Updated Format",
+        "issuer": "Issuer",
+        "contactPerson": "Contact Person",
+        "jobAddress": "Job Address",
+        "jobDate": "2023-06-14",
+        "startTime": "10:00:00",
+        "endTime": "12:00:00",
+        "status": null,
+        "jobComment": "Job Comment"
+    }
+    """;
+
+        mockMvc.perform(MockMvcRequestBuilders.put("/api/jobs/" + addedJob.getUuid())
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(updatedJobJson))
+                .andExpect(status().isOk());
+
+        String updatedContent = mockMvc.perform(MockMvcRequestBuilders.get("/api/jobs/" + addedJob.getUuid()))
+                .andExpect(status().isOk())
+                .andReturn()
+                .getResponse()
+                .getContentAsString();
+
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/jobs/" + addedJob.getUuid()))
+                .andExpect(status().isOk())
+                .andExpect(content().json(updatedContent));
+    }
+
+
+
+
+    @Test
+    @DirtiesContext
+    void deleteJob() throws Exception {
+        MvcResult result = mockMvc.perform(MockMvcRequestBuilders.post("/api/jobs")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content("""
+                            {
+                                "jobFormat": "Format",
+                                "issuer": "Issuer",
+                                "contactPerson": "Contact Person",
+                                "jobAddress": "Job Address",
+                                "jobDate": "2023-06-14",
+                                "startTime": "10:00:00",
+                                "endTime": "12:00:00",
+                                "status": null,
+                                "jobComment": "Job Comment"
+                            }
+                        """))
+                .andExpect(status().isCreated())
+                .andReturn();
+        String content = result.getResponse().getContentAsString();
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        Job[] addedJobs = objectMapper.readValue(content, Job[].class);
```

Comment:
Wieso machst du hier einen Job Array draus? Würde hier nicht auch reichen, wenn du einfach...

Job addedJob = objectMapper.readValue(content, Job.class);

Reply:
array is needed because the default output volume of the getter an aray is.